lib/javascript/food/foodTable.js

```diff
@@ -24,34 +24,52 @@ class FoodTable{
   listen(){
     this.listenForDelete()
     this.listenForEditFoodName()
+    this.listenForEditFoodCalories()
   }
 
   listenForDelete(){
     $('.manage-foods').on('click', '.delete-icon', function(event){
-      var foodId = event.target.classList[1].split('-').pop();
+      let foodId = event.target.classList[1].split('-').pop();
       this.deleteFood(foodId);
       $(event.target.parentElement.parentElement).remove()
     }.bind(this))
   }
 
   listenForEditFoodName(){
     $('.manage-foods').on('focusout', '.food-name', function(event){
-      var newFoodName = event.target.innerText
-      var id = event.target.id.split('-').pop();
-      this.editFood(newFoodName, id)
+      let newFoodName = event.target.innerText
+      let id = event.target.id.split('-').pop();
+      this.editFood("name", newFoodName, id)
     }.bind(this))
   }
 
-  editFood(newFoodName, id){
+  listenForEditFoodCalories(){
```

Comment:
@charliecorrigan @MrAaronOlsen 

this method looks pretty similar to your `listenForEditFoodName` - in the future as a place to refactor, you could work on making this code more DRY (not something you need to do right now, obviously).

i like how you've structured this and that you've worked to break down your code into smaller functions 👍